README.md

```diff
@@ -103,6 +103,69 @@ $message->setBody($body);
 
 > For accessibility purposes, you should *always* provide both a text and HTML version of your mails.
 
+### `multipart/alternative` emails with attachments
```

Comment:
I've raised a [PR](https://github.com/laminas/laminas-mail/pull/254) against `laminas/laminas-mail` to fix what I think are issues with their documentation. If that's merged these changes may not be needed.

Reply:
@kynx It seems they are merging your work :). So .. what do you think makes most sense here? Drop this PR, or merge it neverthless?

Reply:
This PR is still required so the text and html can be extracted from the `multipart/alternative` part. But I'll just link to the laminas docs for a (now correct) example of how to create the email in the first place.